api/nodes/serializers.py

```diff
@@ -267,6 +267,18 @@ def create(self, validated_data):
             node.save()
         except ValidationValueError as e:
             raise InvalidModelValueError(detail=e.message)
+        if request.GET.get('inherit_contributors') and validated_data['parent'].has_permission(user, 'write'):
```

Comment:
How does `validated_data['parent'].has_permission` work? Isn't `validated_data['parent']` a primary key? Or does the node get loaded somewhere?